File: frontend/lib/models/auth/login_response.dart
class LoginResponse {
  final String userId;
  final String email;
  final String phone;
  final String fullName;
  final String role;
  final String accessToken;
  final String refreshToken;
  final int expiresIn;

  LoginResponse({
    required this.userId,
    required this.email,
    required this.phone,
    required this.fullName,
    required this.role,
    required this.accessToken,
    required this.refreshToken,
    required this.expiresIn,
  });

  factory LoginResponse.fromJson(Map<String, dynamic> json) {
    return LoginResponse(
      userId: json['userId'] ?? '',
      email: json['email'] ?? '',
      phone: json['phone'] ?? '',
      fullName: json['fullName'] ?? '',
      role: json['role'] ?? 'user',
      accessToken: json['accessToken'] ?? '',
      refreshToken: json['refreshToken'] ?? '',
      expiresIn: json['expiresIn'] ?? 86400,
    );
  }
}

File: backend/src/main/java/com/nhtl/dto/auth/LoginResponse.java
package com.nhtl.dto.auth;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class LoginResponse {
    
    private String userId;  // ✅ STRING, pas UUID!
    private String email;
    private String phone;
    private String fullName;
    private String role;
    private String accessToken;
    private String refreshToken;
    private long expiresIn;
}